Shared/Accessibility.swift

```diff
@@ -4,6 +4,8 @@
 
 import UIKit
 
+// Test
```

Comment:
I'm guessing this was a test and will be removed, just commenting as a reminder 🙂